lib/Models/WebMapServiceCatalogItem.ts

```diff
@@ -459,6 +465,19 @@ class GetCapabilitiesStratum extends LoadableStratum(
     }
   }
 
+  @computed
+  get isThredds(): boolean {
+    if (this.catalogItem.url) {
+      if (this.catalogItem.url.indexOf("thredds") > -1) return true;
```

Comment:
This probably isn't the best check although I've never seen a thredds server without `thredds` in the title